packages/console/components/Console.svelte

```diff
@@ -0,0 +1,72 @@
+<script context="module">
+	export const evidenceInclude = true;
+</script>
+
+<script>
+	import { browser } from '$app/environment';
+	import { query, setData } from './duckdb';
+	import DataTable from './DataTable.svelte';
+
+	/** @type {Record<string, unknown[]>} */
+	export let data;
+	export let sql_query = 'select * from data limit 100';
+
+	let limit = 10;
+	let pagination = 1;
+
+	$: setData('data', data);
+
+	$: total_rows = query(`
+        SELECT COUNT(*) FROM (${sql_query.replace(/;$/, '')})
+    `);
+
+	$: paginated_results = query(`
+	    WITH query as (${sql_query.replace(/;$/, '')})
+        SELECT * FROM query
+	    LIMIT ${limit}
+	    OFFSET ${(pagination - 1) * limit};
+	`);
```

Comment:
We should make sure that this is [debounced](https://www.npmjs.com/package/lodash.debounce)